### DISCUSSION TITLE
ROS2 script commands to UR

### DISCUSSION THREAD

**Discussion by Hytac** [UniversalRobots/Universal_Robots_ROS2_Driver#463] [Q&A] [answered]
Hello, im migrating my application to ROS2 and im facing some problems sending script commands to the UR.
In ROS we used to have something like this:
`command_sub_ = robot_hw_nh.subscribe("script_command", 1, &HardwareInterface::commandCallback, this);`

`void HardwareInterface::commandCallback(const std_msgs::StringConstPtr& msg)`
`if (ur_driver_->sendScript(str))`

So i can simple publish my command into the topic script_command.

Now in ROS2 i can not find anything like this. How do you guys send commands to the UR?
KR

**Comment by fmauch** [comment, marked as answer]
This is currently not (yet) implemented.

Please see https://github.com/UniversalRobots/Universal_Robots_ROS2_Driver/issues/251 for that.

**Reply by fmauch** [reply]
I'll lock this discussion to keep everything for that inside the issue.
